setup.py

```diff
@@ -19,6 +19,26 @@
 from setuptools import setup
 from setuptools.extension import Extension
 
+import subprocess
+
+compile_time_env = {"LIBSYSTEMD_VERSION": -1}
```

Comment:
i wonder if it make sense to allow the user to override this... probably check for the value of an env variable like PYSTEMD_LIBSYSTEMD_VERSION.

also maybe set the default value to 500, as a pun on the D of systemd (D=500)